tests/crypto_policy_test.go

```diff
@@ -196,41 +217,62 @@ func tryToAccessEndpoint(pod core.Pod, subpath string, port uint16, tlsConfig cl
 		subpath = "/" + subpath
 	}
 
-	// ${webhookServiceName}.${deploymentNamespace}.svc is used to access the endpoints we are testing.
-	// It is used here for the metrics as well for simplicity, as it is the CN in the ca_cert
-	// and the metrics just sit on a different port on the same pod.
-	attemptedUrl = fmt.Sprintf("https://%s.%s.svc:%d%s", strategy.GetSSPWebhookServiceName(), strategy.GetSSPDeploymentNameSpace(), port, subpath)
+	// ${serviceName}.${serviceNamespace}.svc is used to access the endpoints we are testing.
```

Comment:
nit: You've mentioned `${serviceName}.${serviceNamespace}` but instead do `${serviceName}.${pod.Namespace}`.

Reply:
Yes. I think the Service must be in the same namespace as Deployment and Pods. It is possible to point Service to a pod in different namespace, but not with simple selector that we use.
https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors

Reply:
Okay :+1: